### DISCUSSION TITLE
Quiet ::debug:: log output

### DISCUSSION THREAD

**Discussion by kylegibbons** [nektos/act#1441] [Q&A] [answered]
Is there a way to turn off log output that has ::debug:: in the output? Even with the quiet flag this still appears, and it makes it hard to find the log lines I actually care about.

**Comment by KnisterPeter** [comment, marked as answer]
That's currently not possible. Feel free to raise a feature request or even better provide a PR for that.

**Reply by lawndoc** [reply]
I'm interested in taking a look at this. Do you have any advice on where to start? I'm not familiar with the codebase yet.
